apidoc2swagger/apidoc2swagger.php

```diff
@@ -162,6 +186,27 @@ private function createResponseDefinitions($_apiCall)
         $this->swaggerArray["definitions"][$name]["properties"] = $definitions;
         $this->swaggerArray["definitions"][$name]["required"] = $required;
     }
+
+    /**
+     * Searches the operationidmapping.txt file for a "camelCased" operationId.
+     * This file contains mapping for the operation urls like so: /core/renameormove=renameOrMove.
+     *
+     * @param String $_url The ulr of an operation. E.g. /core/renameormove
+     * @return mixed The "camelCased" operationId for the given url. E.g. renameOrMove
+     * @throws Exception when $_url is missing in operationidmapping.txt
+     */
+    private function getOperationId($_url)
+    {
+        $lines = file(__DIR__."/operationidmapping.txt");
```

Comment:
Please, please, **PLEASE** cache this.
And use `parse_ini_file()` instead which makes it so much easier!

```php
private function getOperationId($_url)
{
  if (!$this->operationMapping) $this->operationMapping=parse_ini_file(__DIR__."/operationidmapping.txt");
  if (isset($this->operationMapping[$_url]) return $this->operationMapping[$_url];
  else throw new Exception('The URL $_url is missing in operationidmapping.txt!');
}
```